examples/vanilla-cdn/README.md

```diff
@@ -0,0 +1,13 @@
+# vanilla cdn
+
+## Project setup
+
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
```

Comment:
I would rephrase this to "Starts a local server with hot-reloads for development", since there's no compiling happening.

Reply:
done